src/ItemAntivirus.php

```diff
@@ -329,64 +316,53 @@ private static function showForItem(CommonDBTM $asset, $withtemplate = 0)
-                } else {
-                    echo "</td>";
-                }
-                echo "<td>" . $data['antivirus_version'] . "</td>";
-                echo "<td>" . $data['signature_version'] . "</td>";
-                echo "<td>" . Dropdown::getYesNo($data['is_active']) . "</td>";
-                echo "<td>" . Dropdown::getYesNo($data['is_uptodate']) . "</td>";
-                echo "<td>" . Html::convDate($data['date_expiration']) . "</td>";
-                echo "</tr>";
-                Session::addToNavigateListItems(__CLASS__, $data['id']);
-            }
-            echo $header;
-        } else {
-            echo "<tr class='tab_bg_2'><th colspan='$colspan'>" . __('No item found') . "</th></tr>";
-        }
+        TemplateRenderer::getInstance()->display('components/form/item_antivirus_item.html.twig', [
+            'canedit'               => ($canedit && !(!empty($withtemplate) && ($withtemplate == 2))),
+            'has_antivirus'         => ($result->numrows() != 0),
+            'antivirus_form_url'    => ItemAntivirus::getFormURL() .
+                                        "?itemtype=$itemtype&items_id=$ID&withtemplate=" . $withtemplate,
+        ]);
 
-        echo "</table>";
-        echo "</div>";
+        $antivirus = new self();
+        $entries = [];
+        foreach ($result as $data) {
+            $antivirus->getFromDB($data['id']);
+            $manufacturer = new Manufacturer();
+            $manufacturer->getFromDB($data['manufacturers_id']);
+            $entries[] = [
+                'name'          => $antivirus->getLink(),
+                'is_dynamic'    => Dropdown::getYesNo($data['is_active']),
+                'manufacturers_id' => $manufacturer->getLink(),
+                'antivirus_version' => $data['antivirus_version'],
+                'signature_version' => $data['signature_version'],
+                'is_active'     => Dropdown::getYesNo($data['is_active']),
+                'is_uptodate'   => Dropdown::getYesNo($data['is_uptodate']),
+                'date_expiration' => Html::convDate($data['date_expiration']),
+            ];
+        }
+        TemplateRenderer::getInstance()->display('components/datatable.html.twig', [
+            'is_tab' => true,
+            'nopager' => true,
+            'nofilter' => true,
+            'nosort' => true,
+            'columns' => [
+                'name' => __('Name'),
+                'is_dynamic' => __('Automatic inventory'),
+                'manufacturers_id' => Manufacturer::getTypeName(1),
+                'antivirus_version' => __('Antivirus version'),
+                'signature_version' => __('Signature database version'),
+                'is_active' => __('Active'),
+                'is_uptodate' => __('Up to date'),
+                'date_expiration' => __('Expiration date'),
+            ],
+            'formatters' => [
+                'name' => 'raw_html',
+                'manufacturers_id' => 'raw_html',
```

Comment:
`date_expiration` should use the `date` formatter (or `datetime` if it includes time).